src/idiom263.fuzion

```diff
@@ -0,0 +1,2 @@
+(f64s.log n / f64s.log 2).floor
+(f64s.log n / f64s.log 2).ceil
```

Comment:
This does not match the specs: _Write two functions log2d and log2u,..._, these two functions are missing.